source/uwp/Renderer/lib/XamlHelpers.cpp

```diff
@@ -601,9 +628,9 @@ namespace AdaptiveCards::Rendering::Uwp::XamlHelpers
 
         EventRegistrationToken clickToken;
         // Add Tap handler that sets the event as handled so that it doesn't propagate to the parent containers.
-        return uiElement->add_Tapped(Callback<ITappedEventHandler>([](IInspectable* /*sender*/, ITappedRoutedEventArgs* args) -> HRESULT {
-                                         return args->put_Handled(TRUE);
-                                     }).Get(),
+        return uiElement->add_Tapped(Callback<ITappedEventHandler>([](IInspectable* /*sender*/, ITappedRoutedEventArgs* args) -> HRESULT
+                                                                   { return args->put_Handled(TRUE); })
+                                         .Get(),
```

Comment:
is this formatting from clangformat? I'm not sure I like it...

Reply:
Yep, that's the auto-formatting as applied by Visual Studio. The auto-formatting has always seemed to me to be a little weird and inconsistent around lambdas.